t/plugin/response-rewrite.t

```diff
@@ -0,0 +1,325 @@
+#
```

Comment:
no test case for remove header.

Reply:
ok,I will add e2e test to check the existence of a deleted header，which already set from upstream

Reply:
@lilien1010  do you add test cases? and I think add test case in this file is ok, not need e2e.

Reply:
@lilien1010 any update?